examples/8a_benchmark_write_parallel.cpp

```diff
@@ -310,9 +310,16 @@ class TestInput
 
   int m_MPISize = 1; //!< MPI size
   int m_MPIRank = 0; //!< MPI rank
+
   unsigned long m_XBulk = 64ul; //!< min num of elements at X dimension
   unsigned long m_YBulk = 32ul;  //!< min num of elements at Y dimension
   unsigned long m_ZBulk = 32ul;
+
+  /**  relative expansion of min grid(m_XBulk, m_YBulk, m_ZBulk)
+   *   to form a max grid. by fault max:min=1
```

Comment:
I don't fully understand this part yet, but maybe the additional doc will solve that.

Reply:
Added more comments there. Mainly this option is to direct range of workload per rank. Maybe the name is not fitting really well.